megatron/model/rmsnorm.py

```diff
@@ -1,16 +1,26 @@
+# Copyright (C) 2024 Habana Labs, Ltd. an Intel Company.
+
+from deepspeed.accelerator import get_accelerator
+from megatron import get_args
+
 import torch
+from torch.nn import init
 from torch.nn.parameter import Parameter
 
 # Taken from facebookresearch/llama
 class RMSNorm(torch.nn.Module):
     def __init__(self, dim: int, eps: float = 1e-6):
         super().__init__()
         self.eps = eps
-        self.weight = Parameter(torch.ones(dim))
+        self.weight = Parameter(torch.empty(dim,
```

Comment:
Ditto, creating device tensors instead of host tensors?

Reply:
eventually while running the model on device weights are moved

Reply:
please review now